### ISSUE TITLE
Worms Crazy Golf (70620)

### ISSUE THREAD

**Issue by grabmateusz** [ValveSoftware/Proton#1416]
# Compatibility Report
- Name of the game with compatibility issues: Worms Crazy Golf
- Steam AppID of the game: 70620

## System Information
- GPU: GeForce GT 540M
- Driver/LLVM version: nvidia 390.48
- Kernel version: 4.15.0-33-generic
- [Link to full system information report](https://gist.github.com/grabmateusz/9df08b1e0ec5ab3cf81fa4b8431c3bca)
- Proton version: 3.7-6 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[Game log](https://gist.github.com/grabmateusz/d6881513dcc2887021a646e9d152ae21)

## Symptoms

General feeling:
Game runs very well. I've played through 8 holes in Britannia. All the achievements kicks in. Works nice with Steam Controller.

### Issues:
1) The game does not have any sound. I've googled a bit and apparently [this issue is well known](https://steamcommunity.com/app/70620/discussions/2/34094415776635336/)
2) The game does not start when run for the first time. Second "play" hit starts the game.

### Solution:
1) Install xact. [Apparently this is issue that is not related only to this game](https://github.com/ValveSoftware/Proton/issues/54).
WINEPREFIX=~/.steam/steam/steamapps/compatdata/70620/pfx/ winetricks xact
2) No solution found.

## Reproduction
Run the game :)

**Comment by flibitijibibo** [comment]
Digging this old thing up - is there a folder somewhere for this game that includes .xgs/.xsb/.xwb files?

**Comment by ottopower** [comment]
> Digging this old thing up - is there a folder somewhere for this game that includes .xgs/.xsb/.xwb files?

Yes, in the folder Default/Win there are all these xact files. Now the game works perfectly with 3.16-6 beta, this issue can be closed.
